src/inc/msquic_winuser.h

```diff
@@ -139,7 +151,7 @@ inline
 BOOLEAN
 QuicAddrIsValid(
     _In_ const QUIC_ADDR* const Addr
-    )
+)
```

Comment:
The indented formatting here was correct. Same with all the formatting changes below.

Reply:
Is there a clang format file in place? I have clang format on save on in VS2019 so these changes were unintended, I can revert them